stream_processor.py

```diff
@@ -0,0 +1,32 @@
+from picamera import PiCamera
+#import cv2
+import io
+import time
+
+
+class StreamProcessor:
+
+    def __init__(self, operation: str):
+        self.IMG_H = 720
+        self.IMG_W = 1280
+        self.fr_calc = 1
+        self.sample_size = 2
+        self.export_image_name = f'{operation}_image.jpg'
+
+    # Get frame and drop to image.jpeg
+    def process_stream(self):
+        cam = PiCamera()
+        cam.resolution = (self.IMG_W, self.IMG_H)
+        cam.framerate = 25
+        for sample in range(self.sample_size):
```

Comment:
Optimize loop to post process video stream rather than capture images off stream at some point